media-sound/wildmidi/wildmidi-0.4.4.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake readme.gentoo-r1
+
+DESCRIPTION="MIDI processing library and player using the GUS patch set"
+HOMEPAGE="http://www.mindwerks.net/projects/wildmidi/"
+SRC_URI="https://github.com/Mindwerks/${PN}/archive/${P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~ppc64 ~x86"
+IUSE="+alsa openal oss +player"
+
+DEPEND="
+	player? (
+		alsa? ( media-libs/alsa-lib )
+		openal? ( media-libs/openal )
+	)
+"
+RDEPEND="${DEPEND}
+	media-sound/timidity-freepats
+"
+
+REQUIRED_USE="player? ( ^^ ( alsa oss openal ) )"
```

Comment:
I'd like to transform this `REQUIRED_USE` into a priority-like contsruct within `src_configure`. Something like, if `+alsa` then choose alsa, if `-alsa +oss` choose oss and if `-alsa -oss +openal` use that.